### PR TITLE
USWDS-Site: Add February 2024 monthly call

### LINKED ISSUE #2529: Add Feb. video to USWDS monthly calls page
Purpose
To add the Feb. monthly call video and a brief description of the video to the [USWDS monthly calls page](https://designsystem.digital.gov/about/monthly-calls/).

Steps to be completed

- [ ]  Draft copy for monthly calls page
- [ ]  Get feedback from content team
- [ ]  Get approval from fed team
- [ ]  Create branch to update YML page
- [ ]  Submit pull request
- [ ]  Pull request approval

### LINKED ISSUE #2551: USWDS-Site - Enhancement: Add GitHub Q&As to monthly calls page
Starting in Feb. 2024, we began adding a GitHub discussion post about Q&As from each monthly call. Here is [the February Q&A post](https://designsystem.digital.gov/about/monthly-calls/).

Going forward, we would like to add these Q&As as a bullet point under each month's listing on the [Monthly calls page](https://designsystem.digital.gov/about/monthly-calls/) of the USWDS website. We will not be doing this retroactively--only in Feb. 2024 and subsequent months.

This issue assigns the USWDS dev team the work of deciding on a mechanism for adding this new piece of content to the MC page.

### PR DESCRIPTION
**Pull request by finekatie**
Adding the February event, along with a description and related links, to the USWDS monthly call page.

# Summary

This purpose of this PR is to update the monthly call page to include a video (and supplemental information) for the February 2024 monthly call. This update will keep the page relevant and timely.

> [!IMPORTANT]
> This PR is related to #2631 as we now have updates for March as well. This PR should be merged before that one. 

## Related issue

Closes #2551
Closes #2529

## Preview link

[Monthly calls page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/kf-feb-24-mc-page-updates/about/monthly-calls/)

## Problem statement

The February 2024 event needs to be added to the USWDS monthly calls page.

## Solution

This action will add the February 2024 event to the monthly call page so that the USWDS community can enjoy it!

## Major changes

n/a

## Testing and review

Please proof the new content and make sure it's formatted correctly.

### CODE REVIEW

**Comment by finekatie** [comment]
Added February monthly call copy to the yml file. Needs review for line 7 = ID. Not sure how that is assigned. Needs review for adding Q&A link. I put in the full link title and link, but that's different from how video and event link were added previously. We won't have video for at least another week.

**Comment by amyleadem** [comment]
This afternoon I'll work on updating the code here to include the Q&A link and also remove the video preview if no ID is provided.

**Comment by finekatie** [comment]
> This afternoon I'll work on updating the code here to include the Q&A link and also remove the video preview if no ID is provided.

Oh does the id come from the video? We won't have that for at least another week.

**Comment by amyleadem** [comment]
@finekatie @sarah-sch I've updated the code do hide the video for now and add the Q&A copy. Let me know if that looks good to you!

**Comment by finekatie** [comment]
> @finekatie @sarah-sch I've updated the code do hide the video for now and add the Q&A copy. Let me know if that looks good to you!

Looks good to me! Thank you, Amy!

**Review by amyleadem — APPROVED** [review]
LGTM! We can add the video once it is ready.

**Review by annepetersen — APPROVED** [review]
LGTM

**Comment by finekatie** [comment]
We got the link for the video today. I just added the id for it and committed.

**Comment by annepetersen** [comment]
@thisisdano I've looked at this: could we publish it without your review?

**Comment by thisisdano** [comment]
Note that I removed all the `affectsGuidance` keys in the changelogs, since this content isn't guidance

File: _data/changelogs/about-monthly-calls.yml [hunk 1]
```diff
@@ -2,8 +2,19 @@ title: Monthly calls
 type: documentation
 changelogURL:
 items:
+  - date: NNNN-NN-NN
```
Comment (thisisdano) [inline _data/changelogs/about-monthly-calls.yml:5]:
```suggestion
  - date: 2024-04-26
```

**Review by thisisdano — APPROVED** [review]

**Comment by finekatie** [comment]
Hey @amyleadem , can you take a look at this? Dan approved, but it's failing checks.

**Pull request merged**